src/model.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) const BOOTUPD_UPDATES_DIR: &str = "usr/lib/bootupd/updates";
 #[derive(Serialize, Deserialize, Clone, Debug, Hash, PartialEq, Eq)]
 pub(crate) struct ContentMetadata {
     /// The timestamp, which is used to determine update availability
-    pub(crate) timestamp: NaiveDateTime,
+    pub(crate) timestamp: DateTime<Utc>,
```

Comment:
As this holds content which should stay stable across time, it would be better to use a plain-old type here (i64) and just note that it is a Unix-epoch timestamp.